doc/articles/guides/status-bar-theme-color.md

```diff
@@ -0,0 +1,149 @@
+    ```cs
+    public App()
+    {
+        ConfigureFilters(global::Uno.Extensions.LogExtensionPoint.AmbientLoggerFactory);
+
+    #if __IOS__ || __ANDROID__
+        // Enable native frame navigation.
+        Uno.UI.FeatureConfiguration.Style.ConfigureNativeFrameNavigation();
+    #endif
+
+        this.InitializeComponent();
+        this.Suspending += OnSuspending;
+    }
+    ```
+
+6. Subscribe to the `UISettings.ColorValuesChanged` event from `App.xaml.cs`:
+    > Note that the instance of `UISettings` is kept to prevent it from being disposed when going out of scope, which would otherwise dispose the event subscription.
+    ```cs
+    public sealed partial class App : Application
+    {
+        private UISettings _uiSettings;
+        private bool? _wasDarkMode = null;
+
+        /* ... */
+
+        protected override void OnLaunched(LaunchActivatedEventArgs e)
+        {
+            /* existing code... */
+
+            // add this line at the end of OnLaunched:
+            ConfigureStatusBar();
+        }
+
+        /* ... */
+
+        private void ConfigureStatusBar()
+        {
+            // Listen for the system theme changes.
+            _uiSettings = new UISettings();
+            _uiSettings.ColorValuesChanged += (s, e) => UpdateStatusBar();
+
+    #if __IOS__
+            // Force an update when the app is launched.
+            UpdateStatusBar();
+    #endif
+        }
+    }
+    ```
+
+7. Implement the `UpdateStatusBar()` method:
+    ```cs
+    private void UpdateStatusBar()
+    {
+        // === 1. Determine the current theme from the background value,
+        // which is calculated from the theme and can only be black or white.
+        var backgroundColor = _uiSettings.GetColorValue(UIColorType.Background);
+        var isDarkMode = backgroundColor == Windows.UI.Colors.Black;
+
+    #if __ANDROID__
+        // Prevent deadlock as setting StatusBar.ForegroundColor will also trigger this event.
+        if (_wasDarkMode == isDarkMode) return;
+        _wasDarkMode = isDarkMode;
+    #endif
+
+    #if __IOS__ || __ANDROID__
+        // === 2. Set the foreground color.
+        // note: The foreground color can only be set to a "dark/light" value. See uno remarks on StatusBar.ForegroundColor.
+        // note: For ios in dark mode, setting this value will have no effect.
+        var foreground = isDarkMode ? Windows.UI.Colors.White : Windows.UI.Colors.Black;
+        Windows.UI.ViewManagement.StatusBar.GetForCurrentView().ForegroundColor = foreground;
+
+        // === 3. Set the background color.
+        var background = isDarkMode ? Windows.UI.Colors.MidnightBlue : Windows.UI.Colors.SkyBlue;
+    #if __ANDROID__
+        // On Android, this is done by calling Window.SetStatusBarColor.
+        if (Uno.UI.ContextHelper.Current is Android.App.Activity activity)
+        {
+            activity.Window.SetStatusBarColor(background);
+        }
```

Comment:
Shouldn't we do this in `Windows.UI.ViewManagement.StatusBar.GetForCurrentView().ForegroundColor` instead ?

Reply:
Did you mean with `StatusBar.BackgroundColor`? Currently, it is just not implemented.